include/lsst/afw/image/VisitInfo.h

```diff
@@ -89,6 +89,7 @@ class VisitInfo : public table::io::PersistableFacade<VisitInfo>, public typehan
      * @param[in] id  Identifier of this full focal plane data.
      * @param[in] focusZ Defocal distance of main-cam hexapod in mm. 0 is in focus.;
      *                   Extra-focal is negative while intra-focal is positive.
+     * @param[in] observationType Type of this observation (e.g. science, dark, flat, bias, focus).
```

Comment:
open-ended. Butler can't enforce specific observation types.

Reply:
I don't see how the Butler is relevant here.

My question was motivated by the fact that, if this is going into `VisitInfo`, then it may be parsed by downstream code (otherwise why have it?). Establishing whether or not there are conventions, and if so what they are, is important for avoiding bugs.

Reply:
Currently butler is the main user but I really meant that the metadata translator can't know every single observation type because instruments have many different modes of operation. You can't know the full list ever because there is no global data model of allowed observing modes. "dark" and "bias" are fairly standard but DECam has modes that HSC doesn't have and writing them all down is impossible. You have to refer to the specific instrument's documentation. Each metadata translator can know the obs types for its instrument but that's the limit.